virtual/registry/kv_registry.go

```diff
@@ -431,15 +434,28 @@ func (k *kvRegistry) Heartbeat(
 
 		var state serverState
 		if !ok {
+			serverVersion = 0
```

Comment:
Using an integer that starts at 0 seems a bit risky if we ever add a mechanism to GC old server IDs 🤔 but maybe its ok for now

Reply:
makes sense. I changed it to start from 1